pkg/querier/tail.go

```diff
@@ -8,6 +8,7 @@ import (
 	"github.com/cortexproject/cortex/pkg/util"
 	"github.com/go-kit/kit/log/level"
 	"github.com/grafana/loki/pkg/iter"
+	legacy "github.com/grafana/loki/pkg/loghttp/legacy"
```

Comment:
This is imported as just `loghttp` in the `tail_test.go` file. Do you want to keep this as legacy?

Reply:
Good call.  Fixing.